conf.yaml

```
@@ -717,8 +717,8 @@ contents:
             prefix:     en/cloud-enterprise
             tags:       CloudEnterprise/Reference
             subject:    ECE
-            current:    release-ms-41
-            branches:   [ {'release-ms-41': 2.6}, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.1, 1.0 ]
+            current:    ms-47
+            branches:   [ {'ms-47': 2.7}, 2.6, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.1, 1.0 ]
```

Is there going to be a `2.6` branch that we can use to continue to make changes to the docs for that version? If not, we might need to leave it like:

```suggestion
            branches:   [ {'ms-47': 2.7}, {'release-ms-41': 2.6}, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.1, 1.0 ]
```

The key is the branch name, and the value is the version name as it shows up in the URL and pages.

---

Ahh, good point. I hadn't considered this when I made this change, thank you!